src/core.js

```diff
@@ -51,11 +51,12 @@ function createModule( name, testEnvironment, modifiers ) {
 		childModules: [],
 		suiteReport: new SuiteReport( name, parentSuite ),
 
-		// Pass along `skip` property from parent module, in case there is one,
-		// to childs. And use own otherwise.
+		// Pass along `skip` and `todo` properties from parent module, in case
+		// there is one, to childs. And use own otherwise.
 		// This property will be used to mark own tests and tests of child suites
-		// as `skipped`.
-		skip: parentModule !== null && parentModule.skip || modifiers.skip
+		// as either `skipped` or `todo`.
+		skip: parentModule !== null && parentModule.skip || modifiers.skip,
+		todo: parentModule !== null && parentModule.todo || modifiers.todo
```

Comment:
I'm pretty sure that with this setup you could wind up with a module that has both `skip` and `todo`, which seems like it could potentially cause problems. Can we update the logic to ensure only one of them is set to `true`?